projects/transformers/tests/sparse_bert_model_test.py

```diff
@@ -32,14 +32,15 @@
     AutoModelForSequenceClassification,
 )
 
-sys.path.insert(0, expanduser("~/nta/nupic.research/projects/transformers")) # noqa
 from models import (
     SparseEmbeddings,
     StaticSparseEncoderBertConfig,
     StaticSparseEncoderBertForMaskedLM,
     StaticSparseEncoderBertForSequenceClassification,
 )
 from nupic.torch.modules import SparseWeights
+
+sys.path.insert(0, expanduser("~/nta/nupic.research/projects/transformers")) # noqa
```

Comment:
I know it is a not a new addition to the code, just moving the line - but it is best not to add absolute path in the codebase, as this will change from user to user. We should see if we can find another option.

Reply:
You should be able to use `sys.path.insert(0, abspath("../"))`